### DISCUSSION TITLE
Relations, an additional condition

### DISCUSSION THREAD

**Discussion by Gsom1** [symfony/symfony#49597] [Q&A] [answered]
Hi!
Could you advise please,
is it possible to specify some additional wheres in relation ManyToOne,
for an instance:

```
class User {
    #[ORM\OneToMany(mappedBy: 'user', targetEntity: Chick::class)]
    private Collection $chicks;
}
```
I want not all chicks but having health > 0

**Comment by MatTheCat** [comment, marked as answer]
Hi, you could use a [filter](https://www.doctrine-project.org/projects/doctrine-orm/en/2.14/reference/filters.html), or expose a dedicated getter which would filter the whole chicks collection.
